tools/ci/docker/ydoc-server-polyglot/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:latest
```

Comment:
Would it make sense to shrink the docker image a bit? Can you use something smaller than `ubuntu`? There are reports of successful execution on top of [Alpine Linux](https://github.com/oracle/graal/issues/386) docker image. With Alpine the Graal.js based image could be smaller than the node.js one bringing immediate benefits during the cloud execution.

Reply:
What are the current sizes of both docker images?

Reply:
> What are the current sizes of both docker images?

```
$ docker images
REPOSITORY             TAG       IMAGE ID       CREATED        SIZE
ydoc-server-nodejs     latest    c274936efaac   19 hours ago   228MB
ydoc-server-polyglot   latest    62d560d1ec9b   21 hours ago   596MB
```

> There are reports of successful execution on top of Alpine Linux docker image.

Currently, the `ydoc` binary does not work with Alpine because of the libc differences. I can try to build a static executable and see if it works.